python/packages/autogen-core/src/autogen_core/application/_utils.py

```diff
@@ -0,0 +1,3 @@
+GRPC_IMPORT_ERROR_STR = (
+    "Distributed runtime features require additional dependencies. Install them with: pip install autogen-core[grpc]"
```

Comment:
We need to update the distributed runtime documentation too. There we may need to include the installation instruction that include the `autogen-core[grpc]==0.4.0.dev6`